en/flight_controller/beaglebone_blue.md

```diff
@@ -65,102 +58,88 @@ Currently running on a:
 
-3. install from source
-```sh
-    git clone https://github.com/StrawsonDesign/librobotcontrol.git
-    cd librobotcontrol
-    sudo make install
-```
+1. Use the one pre-installed in BeagleBoard images.
+1. Install from debian package or repository:
+   ```sh
+   sudo apt update && sudo apt install librobotcontrol
+   ```
+1. Install from source
+   ```sh
+   git clone https://github.com/StrawsonDesign/librobotcontrol.git
+   cd librobotcontrol
+   sudo make install
+   ```
 
 After acquiring the pre-built library,
 
-* select librobotcontrol installation directory, and define it as LIBROBOTCONTROL_INSTALL_DIR environment variable so that other unwanted headers will not be included
-
-* install robotcontrol.h and rc/* into $LIBROBOTCONTROL_INSTALL_DIR/include
-
- install pre-built native (ARM) version of librobotcontrol.* into $LIBROBOTCONTROL_INSTALL_DIR/lib
+1. Select the *librobotcontrol* installation directory, and set it in the `LIBROBOTCONTROL_INSTALL_DIR` environment variable so that other unwanted headers will not be included
+1. Install **robotcontrol.h** and __rc/*__ into `$LIBROBOTCONTROL_INSTALL_DIR/include`
+1. Install pre-built native (ARM) version of librobotcontrol.* into `$LIBROBOTCONTROL_INSTALL_DIR/lib`
 
-then the BeagleBone Blue target can be built on both cross compile host system 
-and native build system, i.e., 
+At this point the BeagleBone Blue target can be built on both cross compile host system and native build system, i.e., 
 ```sh
-    make posix_bbblue_cross [upload]
+make posix_bbblue_cross [upload]
 ```
 
-### File Transfer from the Development Computer to the Target Board
+## Cross Compiler Build
 
-In PX4 cross compile build system, rsync is used to transfer files from the 
-development computer to the target board over a network (WiFi or Ethernet). 
-For rsync over SSH with key authentication, setup SSH using steps similar 
-to ones described [here](../flight_controller/raspberry_pi_navio2.md).
+The recommended way to build PX4 for *BeagleBone Blue* is to compile on a development computer and upload the PX4 executable binary directly to the BeagleBone Blue. 
 
-On development host computer, define BeagleBone Blue board as BBBluePX4 in 
-/etc/hosts, and run the following command to build and upload files:
-```sh
-    make posix_bbblue_cross upload
-```
+> **Tip** This approach is recommended over [native build](#native_builds) due to speed of deployment and ease of use.
 
-To test the uploaded files on BeagleBone Blue board, run the following commands:
 
-```sh
-cd /home/debian/px4 
-sudo ./bin/px4 -s px4.config 
-```
-Currently librobotcontrol requires root access.
-
-Note that if you use a PX4 version before [Posix Shell](https://github.com/PX4/Firmware/pull/10173) 
-is available, run the following commands instead:
+1. First set up *rsync* (this is is used to transfer files from the development computer to the target board over a network - WiFi or Ethernet). 
```

Comment:
@UAV-Pilot YOu will note that I removed this from a top level topic because it read as something you only needed to do for cross compiler build. 

Is that correct? Presumably you need to set up SSH for BOTH build because otherwise you can't talk to the BBB to set up autostart etc?

Reply:
Yes, it's only needed for cross compile build. 

Without any setup after OS image installation,  tools like Putty already can talk to BBB over SSH. The SSH setup mentioned here is only needed if user wants to skip typing password in rsync operation, in other words, without the special SSH setup, rsync still works, but requires user to input password each time. So the SSH setup for rsync is optional, and it's for convenience.